src/jit/codegenarm64.cpp

```diff
@@ -5023,11 +5050,39 @@ void CodeGen::genHWIntrinsicUnaryOp(GenTreeHWIntrinsic* node)
     genProduceReg(node);
 }
 
+//------------------------------------------------------------------------
+// genHWIntrinsicCrcOp:
+//
+// Produce code for a GT_HWIntrinsic node with form CrcOp.
+//
+// Consumes two scalar operands and produces a scalar result
+//
+// This form differs from BinaryOp because the attr depends on the size of op2
+//
+// Arguments:
+//    node - the GT_HWIntrinsic node
+//
+// Return Value:
+//    None.
+//
 void CodeGen::genHWIntrinsicCrcOp(GenTreeHWIntrinsic* node)
 {
     NYI("genHWIntrinsicCrcOp not implemented");
 }
 
+//------------------------------------------------------------------------
+// genHWIntrinsicSimdBinaryOp:
+//
+// Produce code for a GT_HWIntrinsic node with form SimdBinaryOp.
```

Comment:
Does the SIMD size matter?

Reply:
No why?

Reply:
Was just wanting to make sure since there will be Vector64 and Vector128 types, and eventually Vector256+ (if SVE is supported).

Reply:
All bets are off on SVE